### PR TITLE
Added bootstrap carousel component and made some updates to index.html

### PR DESCRIPTION
**Pull request by HIGGO36**
What's this PR do?
Adds a new bootstrap carousel widget and fixes the "back-to-top" functionality of the up chevrons used throughout the page.

Where should the reviewer start?
Can begin by inspecting the new carousel's functionality and look on the index.html page and clicking on the up chevrons used throughout.

How should this be manually tested?
Can manually check the new carousel's functionality and look on the index.html page and clicking on the up chevrons used throughout.

Any background context you want to provide?
We are adding this new carousel offering in responses to a client's recent request.

What are the relevant tickets?
n/a

Screenshots (if appropriate)
We added the 'margin: auto' rule to allow for any width to be used and avoid looking like below:

image

So, now any width less than the carousel's container will auto center like below;

image

Also, we are providing a second carousel option displaying images at full width of the container, which applies the 'width: 100%' rule if used.  This will give another choice for our users.

image

Questions:
Is there a blog post?
n/a
Does the knowledge base need an update?
do not think so at this point.

### CODE REVIEW

**Comment by HIGGO36** [comment]
> * Show what the carousel looks like without the changes.
Ok, added images for a few different scenarios to further explain

> * The current picture icon is a circle and the others are oval and should be a circle
Removed the customized styling and left as default.

> * The font-awesome.scss is not needed since the styles are only needed for carousel functionality
Combined rule from this file with the carousel.scss file and removed it.

> * close the other PR on the main repository
This is now closed.

File: kitchensink/index.html [hunk 1]
```diff
@@ -1144,6 +1143,149 @@ <h2 class="mdl-card__title-text"><i class="fa fa-newspaper-o fa-fw " aria-hidden
...
+                </div>
+              </div>
+              <div class="item">
+                  <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
+                  <div class="carousel-caption">
+                    <h3>Image #6</h3>
+                  </div>
+                </div>
+                <div class="item">
+                    <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
```
Comment (davezen1) [inline kitchensink/index.html]: Can we download some placeholder images instead?

Comment (HIGGO36) [reply kitchensink/index.html]: Yes, good idea, will send this updated commit soon.

Comment (HIGGO36) [reply kitchensink/index.html]: > Can we download some placeholder images instead?

Ok, placeholder images have been brought in locally to our images folder and added to this PR.

**Pull request merged**